main.py

```diff
@@ -87,11 +87,7 @@
     help='Dry run: do not actually communicate with Metamist or Hail Batch, '
     'instead only print a final config and stages to be run',
 )
-@click.option(
-    '--verbose',
-    'verbose',
-    is_flag=True,
-)
+@click.option('--verbose', 'verbose', is_flag=True)
```

Comment:
Better to add `help='…',` than to reformat this one so that it's different from all the others.

Reply:
Perhaps I was unclear, so I've added a commit showing what I meant.